clang/test/SemaOpenCL/builtins-amdgcn-error-wave64.cl

```diff
@@ -4,13 +4,10 @@
 // RUN: %clang_cc1 -triple amdgcn-- -target-cpu gfx1010 -target-feature -wavefrontsize64 -verify -S -o - %s
 // RUN: %clang_cc1 -triple amdgcn-- -target-cpu gfx1010 -verify -S -o - %s
 
+// expected-no-diagnostics
+
 typedef unsigned long ulong;
 
 void test_ballot_wave64(global ulong* out, int a, int b) {
-  *out = __builtin_amdgcn_ballot_w64(a == b);  // expected-error {{'__builtin_amdgcn_ballot_w64' needs target feature wavefrontsize64}}
-}
-
-__attribute__((target("wavefrontsize64")))
-void test_ballot_wave64_target_attr(global ulong* out, int a, int b) {
```

Comment:
Should keep the target_attr test

Reply:
If I kept the attribute it would result in 
```
(frontend): invalid feature combination: 'wavefrontsize32' and 'wavefrontsize64' are mutually exclusive
```

Reply:
I just got rid of the `+w32` run lines.